src/main/java/com/nasrpi/auth/AuthController.java

```diff
@@ -29,5 +30,15 @@
 	public boolean authenticate() {
```

Comment:
what is this method?

Reply:
This you had written to check if the config file exists or not and to show the appropriate button (login or create password)

Reply:
alright